android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -367,6 +368,23 @@ public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCoun
 			{
 				_firstVisibleItem = firstVisibleItem;
 				_visibleItemCount = visibleItemCount;
+				
+				if (listView.getChildAt(0) != null){
+					int currentY = listView.getChildAt(0).getTop();
+					String scrollingDir = null;
+					if (currentY < lastY) {
+						scrollingDir = "up";
+					} else if (currentY > lastY) {
+						scrollingDir = "down";
+					}
+					if (scrollingDir != null){
+						KrollDict eventArgs = new KrollDict();
+						eventArgs.put("direction", scrollingDir);
+						eventArgs.put("velocity", (lastY-currentY));
+						fProxy.fireEvent(TiC.EVENT_SCROLLING, eventArgs, false);
```

Comment:
onScroll is called many times when the listView is scrolling, this may not be the best way to implement this method. onScrollStateChange may be a better option